lib/profiles.js

```diff
@@ -15,12 +15,15 @@ const KMS_MODULE = 'ssm-v1';
  * Creates a Profile for a given account
  *
  *
- * @param {String} accountId the id of the account to associate with the
- *                           Profile.
+ * @param {string} accountId - Id of the account to associate with the Profile.
+ * @param {string} [didMethod='key'] - Profile ID DID method (key or v1, etc).
+ * @param {string} [didMethodMode='test'] - Optional DID Method ledger mode
+ *   (relevant only for Veres One for the moment, to select testnet or main).
  *
  * @return {Promise<Profile>} resolves to a profile's settings.
  */
-exports.create = async ({accountId} = {}) => {
+module.exports.create = async ({
+  accountId, didMethod = 'key', didMethodMode = 'test'} = {}) => {
```

Comment:
@dlongley should we set `v1` as default did method?

Reply:
@dmitrizagidulin, no, that would require a major release, we can leave it as `key` for now. We'll probably want to force a selection in the future (next major release) instead of having a default.

Reply:
Tracked as #10.